code/__DEFINES/clothing.dm

```diff
@@ -37,3 +37,32 @@
 /// How much integrity does a shirt lose every time we bite it?
 #define MOTH_EATING_CLOTHING_DAMAGE 15
 */
+//Whitelist for the suit storage slot on medical suits
+#define MEDICAL_SUIT_ALLOWED_ITEMS list( \
```

Comment:
Not sure why this is here instead of in __DEFINES/inventory.dm where the rest are. You _should_ be able to just copy how those work I think